homeassistant/components/coolmaster/services.yaml

```diff
@@ -0,0 +1,6 @@
+reset_filter:
+  description: Reset the timer of the filter.
+  fields:
```

Comment:
service targets are the modern way to do this https://developers.home-assistant.io/docs/documenting/yaml-style-guide?_highlight=target#service-targets e.g. 
```
  target:
    entity:
      integration: coolmaster
      domain: climate
```